zanata-war/src/main/webapp-jboss/WEB-INF/web.xml

```diff
@@ -318,4 +318,9 @@
     <!--<welcome-file>home.xhtml</welcome-file>-->
   <!--</welcome-file-list>-->
 
+  <error-page>
+    <error-code>404</error-code>
+    <location>/404.seam</location>
```

Comment:
@djansen-redhat We do now have a rewrite rule forwarding `404.seam` to `404.xhtml`, but it would be better to use `404.xhtml` here.